utility.go

```diff
@@ -141,7 +140,6 @@ func Trace(skip int) string {
 // RandomString creates a random string of a given length.
 // Ref: https://siongui.github.io/2015/04/13/go-generate-random-string/
 func RandomString(strlen int) string {
-	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Good call!

According to the lint check:
```
  Error: SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.  (staticcheck)
```

Reply:
Interesting, thanks for sharing.